src/Helpers/Helper.php

```diff
@@ -837,4 +837,26 @@ public static function canIntegrationUseSync(string $integrationName): bool
 	{
 		return isset(Filters::ALL[$integrationName]['fields']);
 	}
+
+	/**
+	 * Get users Ip address.
+	 *
+	 * @param bool $secure Determine if the function will return normal IP or hashed IP.
+	 *
+	 * @return string
+	 */
+	public static function getIpAddress(bool $secure = false): string
+	{
+		$ip = isset($_SERVER['REMOTE_ADDR']) ? \sanitize_text_field(\wp_unslash($_SERVER['REMOTE_ADDR'])) : ''; // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
You should probably check if Cloudflare is used since they act as a reverse_proxy: https://developers.cloudflare.com/support/troubleshooting/restoring-visitor-ips/restoring-original-visitor-ips/

Also, there might be some other situations where `$_SERVER[HTTP_X_FORWARDED_FOR]` or `$_SERVER["HTTP_X_REAL_IP"]` might be a better fit.

Reply:
tnx will do